sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/utils/AvroUtilsTest.java

```diff
@@ -213,7 +213,7 @@ public void testNullableBeamArrayFieldToAvroField() {
     fields.add(
         new org.apache.avro.Schema.Field(
             "decimal",
-            LogicalTypes.decimal(Integer.MAX_VALUE)
```

Comment:
@reuvenlax this change actually breaks `testFromBeamSchema`, because we cannot generate the correct Avro schema from the Beam schema (it doesn't have fixed precision/scale). What do you think about skipping `testFromBeamSchema`until we can fix it more durably (e.g. by adding a fixed decimal logical type to map to)?